best_practices.rst

```diff
@@ -236,11 +236,11 @@ configuration. You don't need to browse several files created with different
 formats (YAML, XML, PHP): all the configuration is just where you need it and
 it only uses one format.
 
-Don't Use Annotations to Configure the Controller Template
+Don't Use Attributes to Configure the Controller Template
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The ``@Template`` annotation is useful, but also involves some *magic*.
-Moreover, most of the time ``@Template`` is used without any parameters, which
+The ``#[Template]`` attribute is useful, but also involves some *magic*.
+Moreover, most of the time ``#[Template]`` is used without any parameters, which
```

Comment:
The `#Template` attribute in Symfony no longer allows to omit the template, does it?

Reply:
I just did a test, and it's still possible to omit it. This is what is stated in the [documentation](https://symfony.com/bundles/SensioFrameworkExtraBundle/current/annotations/view.html#usage).

Reply:
Did you try the `#Template` attribute from the SensioFrameworkExtraBundle or the one that we will ship in the core with Symfony 6.2?

Reply:
Ah you're right, the new one in core requires that the template is specified!

However, as the new attribute will only be available in 6.2, perhaps we can update the documentation only from this version, WDYT ?

Reply:
Ah yes, you are right. 👍